src/status_im/contexts/profile/contact/view.cljs

```diff
@@ -0,0 +1,30 @@
+(ns status-im.contexts.profile.contact.view
+  (:require [quo.foundations.colors :as colors]
+            [quo.theme :as quo.theme]
+            [react-native.reanimated :as reanimated]
+            [status-im.common.not-implemented :as not-implemented]
+            [status-im.common.scroll-page.view :as scroll-page]
+            [status-im.contexts.profile.contact.header.view :as contact-header]
+            [utils.re-frame :as rf]))
+
+(defn scroll-handler
```

Comment:
this is used?
also consider not swapping the argument order here.
i.e you have
```
(defn some-function [second-arg first-arg]
  (internal-function first-arg second-arg))
```
I prefer to use the same order as the function being wrapper where possible.